pkg/shared/utils/stringscanner/sql_string_scanner_test.go

```diff
@@ -80,3 +81,70 @@ func Test_SQLStringScanner(t *testing.T) {
 		})
 	}
 }
+
+func Test_scanner_Peek(t *testing.T) {
+	type fields struct {
+		tokens []Token
+		pos    int
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   bool
+		want1  *Token
+	}{
+		{
+			name: "return true and token if pos < length of tokens -1",
+			fields: fields{
+				pos: 1,
+				tokens: []Token{
+					{Value: "testToken1"},
+					{Value: "testToken2"},
+					{Value: "testToken3"},
+				},
+			},
+			want: true,
+			want1: &Token{
+				Value: "testToken3",
+			},
+		},
+		{
+			name: "return false and nil if pos < length of tokens -1",
+			fields: fields{
+				pos: 2,
+				tokens: []Token{
+					{Value: "testToken1"},
+					{Value: "testToken2"},
+				},
+			},
+			want:  false,
+			want1: nil,
+		},
+		{
+			name: "return false and nil if pos == length of tokens -1",
+			fields: fields{
+				pos: 1,
+				tokens: []Token{
+					{Value: "testToken1"},
+					{Value: "testToken2"},
+				},
+			},
+			want:  false,
+			want1: nil,
+		},
+	}
+
+	RegisterTestingT(t)
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			s := &scanner{
+				tokens: tt.fields.tokens,
+				pos:    tt.fields.pos,
+			}
+			got, got1 := s.Peek()
```

Comment:
variable names - let them be self explanatory.

Reply:
Thanks for the feedback @Rajagopalan-Ranganathan